src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿
+@using SFA.DAS.EAS.Portal.Client.Types;
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+@{
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+
+    if (Model.HasSingleClosedVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Closed);
+        actionLinkText = "Manage vacancy";
+        numberOfApplications = vacancy.NumberOfApplications;
+    };
+}
+<h3 class="das-panel__heading">Your vacancy</h3>
+<p>@vacancy.TrainingTitle <span class="das-!-display--block das-text--muted">VAC @vacancy.Reference</span></p>
+<dl class="das-definition-list das-definition-list--inline">
+    <dt class="das-definition-list__title">Closing date</dt>
+    <dd class="das-definition-list__definition">@(vacancy.ClosingDate.HasValue ? vacancy.ClosingDate.Value.ToGdsFormat() : "-")</dd>
+
+    @{ if (numberOfApplications.HasValue)
+        {
+            <dt class="das-definition-list__title">Applications</dt>
+            <dd class="das-definition-list__definition">@numberOfApplications.Value</dd>
+        }
+    }
+
+    <dt class="das-definition-list__title">Status</dt>
+    <dd class="das-definition-list__definition">@vacancy.Status</dd>
+</dl>
+<ul class="govuk-list">
+    <li><a href="@Url.EmployerRecruitAction(vacancy.ManageVacancyUrl)" class="govuk-link govuk-link--no-visited-state">@actionLinkText</a> </li>
```

Comment:
It would be safer not to render the `<a>` (and `<li>`) if `HasSinglePendingReviewVacancy` is false.
I'm not sure how an anchor with no text would play in all user agents / screen readers etc..